srcpkgs/removed-packages/template

```diff
@@ -69,6 +69,7 @@ replaces="
  enventor<=1.0.0_2
  epstopdf<=2.27_3
  fontmatrix<=0.6.0.20171228_2
+ freshplayerplugin-0.3.9_9
```

Comment:
this should use `<=` patterns like the other removed packages, otherwise it won't be removed from systems where it was not updated to the latest version before the removal.

Reply:
oh i overlooked this

Reply:
ok now?